site/content/docs/howto/supervisor/configure-supervisor-federationdomain-idps.md

```diff
@@ -0,0 +1,403 @@
+---
+title: Configure Identity Providers (IDPs) on a FederationDomain
+description: Learn how to use one or more identity providers, and identity transformations and policies, on a FederationDomain.
+cascade:
+  layout: docs
+menu:
+  docs:
+    name: IDPs on FederationDomains
+    weight: 20
+    parent: howto-configure-supervisor
+---
+
+This guide explains how to associate one or more external identity providers (IDPs) with a FederationDomain.
+It also details how to configure identity transformations and identity policies for those identity
+providers.
+
+## Prerequisites
+
+This how-to guide assumes that you have already [installed the Pinniped Supervisor]({{< ref "install-supervisor" >}})
+and have already read the guide about how to [configure the Supervisor as an OIDC issuer]({{< ref "configure-supervisor" >}}).
+
+This guide focuses on the use of the `spec.identityProviders` setting on the FederationDomain CRD.
+
+## Summary
+
+External identity providers may be configured in the Supervisor by creating OIDCIdentityProvider,
```

Comment:
I think Josh and I talked about this, I tend to want to see our custom resources always in the mono font in our docs, like `OIDCIdentityProvider`, `ActiveDirectoryIdentityProvider`, `LDAPIdentityProvider`.  Prob a conversation to spin out later, but this is a good docs PR to trigger the conversation.

Reply:
For this PR, feel free to resolve.